.github/workflows/checkstyle.yml

```diff
@@ -0,0 +1,20 @@
+name: CheckStyle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: ~
+
+jobs:
+  checkstyle_job:
+    runs-on: ubuntu-latest
+    name: Run CheckStyle checks
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Check for EasyPost style compliance
+        uses: nikitasavinov/checkstyle-action@master
+        with:
+          # github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Let's determine if this is needed or not, uncomment if it is, remove completely if not.

Reply:
Fixed in 5510fba